lib/teleterm/clusters/cluster_databases.go

```diff
@@ -89,7 +89,7 @@ func (c *Cluster) GetDatabases(ctx context.Context) ([]Database, error) {
 }
 
 // ReissueDBCerts issues new certificates for specific DB access
-func (c *Cluster) ReissueDBCerts(ctx context.Context, user string, db types.Database) error {
+func (c *Cluster) ReissueDBCerts(ctx context.Context, user string, db types.Database, databaseName string) error {
```

Comment:
Nit:
Keep the string  db flags together: 
```suggestion
func (c *Cluster) ReissueDBCerts(ctx context.Context, user, dbName string, db types.Database) error {
```